generators/app/index.js

```diff
@@ -290,13 +300,10 @@ module.exports = class extends BaseGenerator {
                     'src/main/java/package/service/kafka/serializer/EntitySerializer.java.ejs',
                     `${javaDir}service/kafka/serializer/${entity}Serializer.java`
                 );
-
-                producersCpt++;
             }
         });
 
-        kafkaProperties += '\n';
-
+        const kafkaProperties = this.generateKafkaProperties();
```

Comment:
You can put directly `this.generateKafkaProperties()` in the next line

Reply:
Or even better remove lines 306 and 307 and `this.log` the returning value of the `generateKafkaProperties` function

Reply:
i need the variable declaration for the usage in line 310 & 311
```javascript
 this.replaceContent(`${resourceDir}config/application.yml`, kafkaBlockPattern, kafkaProperties);
 this.replaceContent(`${testResourceDir}config/application.yml`, kafkaBlockPattern, kafkaProperties);
```